### PR TITLE
Spring boot part3

### PR DESCRIPTION
**Pull request by bentley-ritson-umuzi**
Added changes as discussed.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@ gitignore/
 node_modules/
 public/
 .vscode/
+package-lock.json/
```
Comment (RuddyN) [inline .gitignore:6]: why is there a package-lock.json file anyways, does it have to commit to this project or can it only exist in your, i see you are the one who added it to the repo,

File: .idea/.gitignore [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```
Comment (RuddyN) [inline .idea/.gitignore:1]: remove all .idea files, as they are your own Intellij configs and not the project you should add .idea into the gitignore file

File: .idea/misc.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (RuddyN) [inline .idea/misc.xml:1]: remove all .idea files, as they are your own Intellij configs and not the project you should add .idea into the gitignore file

File: .idea/modules.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (RuddyN) [inline .idea/modules.xml:1]: same thing for all .idea files

File: content/projects/java-specific/introduction-to-spring-boot/part-3/_index.md [hunk 5]
```diff
@@ -1,3 +1,111 @@
...
+
+```
+
+**Step 4**
+
+Specify all your methods inside the UserServiceInterface then implement all services in the UserServiceImpl. 
+
+**Step 5**
+
+1 - Use the annotation to instantiate the UserService class.
```
Comment (RuddyN) [inline content/projects/java-specific/introduction-to-spring-boot/part-3/_index.md:67]: you forgot to mention that they are doing this in the controller

File: .gitignore (hunk 1 above)
Comment (bentley-ritson-umuzi) [reply .gitignore:6]: Forgot about that my Intelij is set up to .add my settings every time I checkout,

File: content/projects/java-specific/introduction-to-spring-boot/part-3/_index.md [hunk 6]
```diff
@@ -1,3 +1,111 @@
...
+
+	[name] - replaced with actual name
+    }
+}
+
+```
+
+**Step 4**
+
+Specify all your methods inside the UserServiceInterface then implement all services in the UserServiceImpl.
```
Comment (RuddyN) [inline content/projects/java-specific/introduction-to-spring-boot/part-3/_index.md:63]: implement all methods in the UserServiceImpl

File: content/projects/java-specific/introduction-to-spring-boot/part-3/_index.md [hunk 7]
```diff
@@ -1,3 +1,111 @@
...
+
+```
+
+**Step 4**
+
+Specify all your methods inside the UserServiceInterface then implement all services in the UserServiceImpl. 
+
+**Step 5**
+
+1 - Use the annotation to instantiate the UserController.
```
Comment (RuddyN) [inline content/projects/java-specific/introduction-to-spring-boot/part-3/_index.md:67]: 1. do the following in the UserController

**Pull request merged**